### PR TITLE
added route

### PR DESCRIPTION
**Pull request by calebrs**
Added new route that takes the webhook from the FF manager and saves the data.

### CODE REVIEW

File: controllers/streamController.js [hunk 1]
```diff
@@ -35,13 +36,18 @@ const handleNewConnection = async (req, res) => {
 const sendUpdate = (req, res, next) => {
   const updatedFlags = updateFlags(req, [...flags]);
```
Comment (julia-martin) [inline controllers/streamController.js]: Does `updateFlags()` work here if it's not defined until later?

Comment (julia-martin) [reply controllers/streamController.js]: When does `sendUpdate()` get called / who is calling this? I couldn't find it anywhere

**Review by julia-martin — COMMENTED** [review]
Great work! Would love to get a code walkthrough during standup tomorrow morning. I'm just wondering how you were able to access the `flags` variable from inside of `handleNewConnection`?

**Comment by calebrs** [comment]
Originally, I didn't change the streamController but I was able to get to that this morning. I'll show it during standup.

**Pull request closed without merging**